src/app/templates/home.html

```diff
@@ -1,45 +1,26 @@
 <!doctype html>
 <html>
 <head>
-    <script src="../static/js/calc.js"></script>
-    <script src="../static/js/main.js" ></script>
-    <script src="../static/js/home.js" ></script>
+    <script src="../static/js/home.js"></script>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-    <link rel="stylesheet" type="text/css" href="../static/css/base_temp.css">
     <link rel="stylesheet" type="text/css" href="../static/css/home.css">
-
-    <nav class="navbar navbar-expand-lg navbar-light bg-light">
-      <img src="../static/images/hulk-logo.png" id = "brand-image" alt="Hulk Logo" class="navbar-image">
-        <a class="navbar-brand" id="brand" href="#">Project Hulk</a>
-        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
-        </button>
-        <div class="collapse navbar-collapse" id="navbarNav">
-          <ul class="navbar-nav">
-            <li class="nav-item">
-              <a class="nav-link" id="nav_search_button">Search </a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" id="nav_calcalc_button">Calculator</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" id="nav_profile_button">Profile</a>
-            </li>
-          </ul>
-        </div>
-    </nav>
 </head>
 <body>
+
+    <div id="navbar-container"></div>
+
     <div id="welcome">
         <p>Welcome to Project Hulk!</p>
     </div>
+
     <div id="home-links">
       <button type="button" class="btn" id="home_search_button">Search for Excercises</button>
       <button type="button" class="btn" id="home_calcalc_button">Calorie Calculator</button>
       <button type="button" class="btn" id="home_profile_button" >User Page</button>
       <button type="button" class="btn" id="home_login_signup_button">Log In or SIgn Up</button>
     </div>
 
+    <script src="../static/js/main.js"></script>
```

Comment:
because the event listener is on dom content loaded, this can be placed at the top of the html and the script should still run after the document is loaded, right? No reason to change anything, just noticed this line was moved specifically.